### ISSUE TITLE
cam doesn't open

### ISSUE THREAD

**Issue by santaboy2009** [alievk/avatarify-python#554]
cam doesn't open

avatarify) C:\Users\12025\avatarify-python>run_windows.bat
[1679991543.029166] Loading Predictor
C:\Users\12025\miniconda3\envs\avatarify\lib\site-packages\torch\functional.py:478: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  C:\b\abs_bao0hdcrdh\croot\pytorch_1675190257512\work\aten\src\ATen\native\TensorShape.cpp:2895.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
[1679991545.794783] Trying camera with id 0
[1679991549.853061] Trying camera with id 1
[1679991550.886233] Trying camera with id 2

**Comment by JohanAR** [comment]
Same as #547

**Comment by pedrogabriel2135** [comment]
Please stop sending me this

**Comment by JohanAR** [comment]
@pedrogabriel2135 you will have to unsubscribe from the thread on github, since it's the site that's sending you updates. I can see that the email you replied to has a link you can click to unsubscribe.

**Comment by davidlorex** [comment]
Follow this steps as exact. 
1. verify your camera is functioning properly
2. after you have ensure that your cam is actively functioning well , then I suggest you locate the avatarify-python folder and delete the cam.yaml 
3. now  run_windows.bat, this will allow the cam selector run allover . 
You welcome .

**Comment by santaboy2009** [comment]
> Follow this steps as exact.
> 
> 1. verify your camera is functioning properly
> 2. after you have ensure that your cam is actively functioning well , then I suggest you locate the avatarify-python folder and delete the cam.yaml
> 3. now  run_windows.bat, this will allow the cam selector run allover .
>    You welcome .

I am having same problems too, I will try this and I will come back and give feedback, thanks for your contribution.

**Comment by iben1999** [comment]
I can’t even find that file

**Comment by ostronic** [comment]
I AM AVAILABLE TO HELP ANYONE HAVING ISSUES INSTALLING AVATARIFY ON WINDOWS PC.
HMU ON TELEGRAM @Qxchan
Please go straight to the point ‼️

**Comment by solid-cmd** [comment]
anyone figured a way to solve this yet????????

**Comment by solid-cmd** [comment]
stops loading after trying camera with i.d 2

**Comment by SanPatrick01** [comment]
If you have many cam uninstall 
If you have xpression uninstall 
If you have split cam uninstall 

The. You're good to go

**Comment by solid-cmd** [comment]
thanks man

**Comment by drbruno511** [comment]
warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
  
  How do i solve this ? please any help

**Comment by SanPatrick01** [comment]
What type of nvidia GPU ? 

For colab 

Colab Avatarify installation on CPU laptops
https://youtu.be/gfPs3tiXYOk


For gaming laptops 

Avatarify windows installation
https://youtu.be/sylE-uIhNCw

**Comment by ostronic** [comment]
> warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
> 
> How do i solve this ? please any help

drop me a message @fg_daemon on Telegram
